engine/core/src/test/resources/classic-engine.properties

```diff
@@ -1,4 +1,5 @@
 org.pentaho.reporting.engine.classic.core.modules.output.pageable.graphics.debug.PrintPageContents=false
+org.pentaho.reporting.engine.classic.core.allowScriptEvaluation=true
```

Comment:
@peterrinehart shouldn't this be false by default

Reply:
That's a test case resource, not the file included in the final product.

Reply:
make sense